driver-ui/src/main/java/eu/tsystems/mms/tic/testframework/pageobjects/internal/asserts/DefaultUiElementAssertion.java

```diff
@@ -210,6 +210,16 @@ public String createSubject() {
         });
     }
 
+    @Override
+    public BinaryAssertion<Boolean> partiallyVisible() {
```

Comment:
For a easier finding of thease methods via code completion I prefer the following names:
`visibleFull`
`visiblePartial`